src/export/core.ts

```diff
@@ -25,16 +25,24 @@ import { use } from '../extension';
 
 // Import label layout by default.
 // TODO will be treeshaked.
-import {installLabelLayout} from '../label/installLabelLayout';
+import { installLabelLayout } from '../label/installLabelLayout';
 use(installLabelLayout);
 
 
 // Export necessary types
-export {ZRColor as Color, Payload, ECElementEvent} from '../util/types';
-export {LinearGradientObject} from 'zrender/src/graphic/LinearGradient';
-export {RadialGradientObject} from 'zrender/src/graphic/RadialGradient';
-export {PatternObject, ImagePatternObject, SVGPatternObject} from 'zrender/src/graphic/Pattern';
-export {ElementEvent} from 'zrender/src/Element';
+export {
+    ZRColor as Color,
+    Payload,
+    ECElementEvent,
+    HighlightPayload,
+    DownplayPayload,
+    SelectChangedPayload
+} from '../util/types';
+export { LinearGradientObject } from 'zrender/src/graphic/LinearGradient';
+export { RadialGradientObject } from 'zrender/src/graphic/RadialGradient';
+export { PatternObject, ImagePatternObject, SVGPatternObject } from 'zrender/src/graphic/Pattern';
+export { ElementEvent } from 'zrender/src/Element';
+export * from './option';
```

Comment:
I'm not sure about this line.  @pissang Do we miss an `echarts/option` entry or `echarts/all` entry? It seems we can only import them from `echarts/types/dist` folder now. Please take a look.

Reply:
@plainheart, you're right. I was looking for the incorrect type name for the option. 

For example, 

```
interface EChartsOption extends ECBasicOption {
    dataset?: DatasetOption | DatasetOption[];
    aria?: AriaOption;
    title?: TitleOption | TitleOption[];
    grid?: GridOption | GridOption[];
    radar?: RadarOption | RadarOption[];
    polar?: PolarOption | PolarOption[];
    geo?: GeoOption | GeoOption[];
    angleAxis?: AngleAxisOption | AngleAxisOption[];
    radiusAxis?: RadiusAxisOption | RadiusAxisOption[];
    xAxis?: XAXisOption | XAXisOption[];
    yAxis?: YAXisOption | YAXisOption[];
    singleAxis?: SingleAxisOption | SingleAxisOption[];
    parallel?: ParallelCoordinateSystemOption | ParallelCoordinateSystemOption[];
    parallelAxis?: ParallelAxisOption | ParallelAxisOption[];
    calendar?: CalendarOption | CalendarOption[];
    toolbox?: ToolboxComponentOption | ToolboxComponentOption[];
    tooltip?: TooltipOption | TooltipOption[];
    axisPointer?: AxisPointerOption | AxisPointerOption[];
    brush?: BrushOption | BrushOption[];
    timeline?: TimelineOption | SliderTimelineOption;
    legend?: LegendComponentOption | (LegendComponentOption)[];
    dataZoom?: DataZoomComponentOption | (DataZoomComponentOption)[];
    visualMap?: VisualMapComponentOption | (VisualMapComponentOption)[];
    graphic?: GraphicComponentLooseOption | GraphicComponentLooseOption[];
    series?: SeriesOption$1 | SeriesOption$1[];
    options?: EChartsOption[];
    baseOption?: EChartsOption;
}
```

I was looking to import `TooltipOption` type. However, it's already exported as `TooltipComponentOption`, which is a little confusing when looking at the type. 

I will revert this change.